core/config/v2/docs/chains-evm.toml

```diff
@@ -37,6 +37,8 @@ ChainType = 'arbitrum' # Example
 # A re-org occurs at height 46 starting at block 41, transaction is marked for rebroadcast
 # A re-org occurs at height 47 starting at block 41, transaction is NOT marked for rebroadcast
 FinalityDepth = 50 # Default
+# FinalityTag means that the chain supports the finalized block tag when querying for a block. This is a placeholder and hasn't been implemented yet.
```

Comment:
nit: Can you also add the following:
If FinalityTag is set to true for a chain, then FinalityDepth field is ignored.
Finality for a block is solely defined by the finality related tags provided by the chain's RPC API.